app/pages/project/home/index.jsx

```diff
@@ -33,6 +35,15 @@ export default class ProjectHomeContainer extends React.Component {
     }
   }
 
+  fetchAllWorkflows(props = this.props, query = { active: true, fields: 'active,completeness,configuration,display_name' }) {
```

Comment:
Are default params ok to use here? MDN says no support in IE or Opera. https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Functions/Default_parameters